src/stream/src/from_proto/source/trad_source.rs

```diff
@@ -197,7 +196,7 @@ impl ExecutorBuilder for SourceExecutorBuilder {
                             (0..source.columns.len()).collect_vec(),    // eliminate the last column (_rw_offset)
                             None,
                             schema.clone(),
-                            pk_indices,
```

Comment:
This is to fix https://buildkite.com/risingwavelabs/pull-request/builds/33652#018b407f-56a9-455f-bccf-9f0fc8eb6998. Plz help take a look @StrikeW.

Reply:
So for a source that has defined pk,  does `params.pk_indices` contains the index to the primary key or it is empty?

Reply:
Currently `params.pk_indices`, which is derived by optimizer, is empty. So basically this change only ensures executor use the same `pk_indices` as optimizer, the actual content of `pk_indices` doesn't matter, because the `Materialize` node at downstream has correct primary key. (I just want to keep changes minimal)